main.cpp

```diff
@@ -13,75 +13,120 @@ What you create in this project will be used as the basis of Project 5 in the co
     
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Environment Controls
-        1)
-        2)
-        3)
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Infotainment System
-        1)
-        2)
-        3)
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1)
-        2)
-        3)
-    Name 2 things you can do with the:   Seat
-        1)
-        2)
+    Here is an example:
+    Cell Phone
+
+    A Cell Phone is made up of the following 5 properties:
+        Display
+        Memory
+        CPU
+        Radio
+        Applications
+
+    A Cell Phone has 3 things it can do:
+        make a call
+        send a text
+        run an application.
+
+    Notice that I did not use "has a display" or "Has memory" or "has a cpu" as one of the properties of the CellPhone
+    
+    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
+    Instead, I wrote "Display" or "CPU".  These are specific objects or amounts. 
+    
+    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
+    Instead, prefer the other primitive types.
+
+    When you choose your 5 smaller parts, remember that each of these 5 Sub Objects will need 
+    to be defined with 5 primitive properties and 3 actions EACH.  
 */
+
+
+
+/*
+Thing 1) school
+5 properties:
+    1) number of classes
+    2) number of teachers
+    3) number of students
+    4) number of new students each year
+    5) number of graduated students
+3 things it can do:
+    1) teach students
+    2) charge parents
+    3) make students socialize
+ */
+
+/*
+Thing 2) snack factory
+5 properties:
+    1) number of products produced each day
+    2) amount of ingredients ordered each week
+    3) amount of money for ordering ingredients
+    4) number of workers
+    5) number of packaging machines
+3 things it can do:
+    1) produce snacks
+    2) distribute snacks
+    3) recruit workers
+ */
+
+/*
+Thing 3) coffee shop
+5 properties:
+    1) amount of coffee in stock
+    2) number of coffee maker machines
+    3) net sales per day
+    4) number of seats available
+    5) number of cups to wash
+3 things it can do:
+    1) serve coffee
+    2) give customers place to laze around
+    3) sell snacks
+ */
+
+/*
+Thing 4) guitar amp
+5 properties:
+    1) number of speakers
+    2) number of bands in the EQ
+    3) amount of gain
+    4) number of output speaker impedance options
+    5) types of effects
+3 things it can do:
+    1) amplify guitar sound
+    2) equalize guitar tone
+    3) change guitar sound
+ */
+
+/*
+Thing 10) front door
+5 properties:
+    1) door knob
+    2) lock
+    3) hinge
+    4) door frame
+    5) eye viewer
```

Comment:
If you think you can coax 5 properties and 3 actions from each of these properties, go for it.
Otherwise, pick another Macro object for Thing 10) and fill in 5-9 accordingly.